src/node/psbt.cpp

```diff
@@ -24,7 +24,7 @@ PSBTAnalysis AnalyzePSBT(PartiallySignedTransaction psbtx)
 
     result.inputs.resize(psbtx.tx->vin.size());
 
-    const PrecomputedTransactionData txdata = PrecomputePSBTData(psbtx);
+    PrecomputedTransactionData txdata = PrecomputePSBTData(psbtx);
```

Comment:
Why did you make `PrecomputedTransactionData` non-`const`?

Reply:
because I put into it a field which mutates internal state during the LazyInit, which would otherwise not be callable unless I made those fields mutable, which is not a preferred pattern in core.